modules/common/src/main/scala/notebook/front/widgets/Dataset.scala

```diff
@@ -37,7 +36,7 @@ trait DataFrameView {
     override implicit def singleCodec: Codec[JsValue, JsValue] = JsonCodec.idCodec
   }
 
-  private lazy val json: RDD[(String, Long)] = data.toJSON.zipWithIndex.cache
+  private lazy val json: RDD[(String, Long)] = data.toJSON.rdd.zipWithIndex.cache
```

Comment:
@andypetrella this implicit `.cache` call shouldn't be here, right (not related to this PR) ?